packages/clarity-visualize/src/interaction.ts

```diff
@@ -1,5 +1,6 @@
 import { Asset, Constant, PlaybackState, Point, Setting } from "@clarity-types/visualize";
-import { Data, Interaction, Layout } from "clarity-decode";
+import { Data, Layout } from "clarity-js";
+import type { Interaction } from "clarity-decode"
```

Comment:
Very minor: Use Decoded as prefix in all case to be consistent with the other changes you made.